YouVsVirus/Assets/Scripts/Components/EndLevelControllerBase.cs

```diff
@@ -73,19 +48,9 @@ public void Start()
     public virtual void EndLevel()
```

Comment:
I think a different way would be
```suggestion
    public abstract void EndLevel();
```
and remove the function footer.
This way we will get a compiler error when a derived class does not implement EndLevel.

Reply:
Removed EndLevel(), but good idea of course